public/framework/catalog_cache/cache_manager.test.tsx

```diff
@@ -125,7 +125,11 @@ describe('CatalogCacheManager', () => {
     });
 
     it('should return default cache object if cache is not found', () => {
-      const defaultCacheObject = { version: CATALOG_CACHE_VERSION, dataSources: [] };
+      const defaultCacheObject = {
+        version: CATALOG_CACHE_VERSION,
+        dataSources: [],
+        dataSourceMDSId: '',
```

Comment:
It is better to not change older tests to maintain BWC. Rather add new tests for new changes, the older ones should work as is.

Reply:
yes, have updated the cache object now it only takes in the id if its present